Tests/UnitTests/FileServices/FileDialogTests.cs

```diff
@@ -46,9 +46,9 @@ public void DirectTyping_Allowed ()
                      );
 
         // continue typing the rest of the path
-        Send ("BOB");
+        Send ("bob");
```

Comment:
I'm seeing many unit tests changed when the v2 is the one who have to be adapted to make them being passed. Only the one that it's proved that was wrong with the v1 drivers should be fixed.

Reply:
SendKeys is hack city.  It has a signature that makes no sense.  It should simply take single arg `Key`.

ConsoleKey is part of input processing not key processing.

I have reluctantly implemented SendKey into facade where previously it did nothing.  But I have had to map to Key.

This test you cite was already hacky and full of workarounds. Specifically it sent in caps then expected driver to interpret as lower by its asserts.

Reply:
But you don't agree that these tests must have to pass as it's pass with v1 fake driver? Do you meaning v1 fake driver is wrong? I'm really confused, sorry. You have to play with the modifiers keys too. See my PR https://github.com/tznind/gui.cs/pull/183, please. Thanks.

Reply:
> This test you cite was already hacky and full of workarounds. Specifically it sent in caps then expected driver to interpret as lower by its asserts.

With the `Key` class implemente by @tig a upper letter is only considered as upper case if the shift flag is true. That the new rule in the Key class. I tought you already know that.

Reply:
With my PR we can now manipulate upper and lower cases to get the same path names as it's sent. But this is only to manipulate path names. For hot keys we need always to sent as upper case.